### PR TITLE
Apr 1633 assessor moderation tab

### PR DESCRIPTION
**Pull request by MichaelYoung1981**

### CODE REVIEW

File: src/SFA.DAS.ApplyService.Data/AssessorRepository.cs [hunk 1]
```diff
@@ -151,6 +151,44 @@ public async Task<int> GetInProgressAssessorApplicationsCount(string userId)
...
+                        $@"SELECT COUNT(1)
+	                      FROM Apply apply
+	                      WHERE (Assessor1ReviewStatus = @approvedReviewStatus OR Assessor2ReviewStatus = @approvedReviewStatus) AND ModerationStatus <> @completedModerationStatus",
+                        new
+                        {
+                            approvedReviewStatus = AssessorReviewStatus.Approved,
+                            completedModerationStatus = ModerationStatus.Complete
+                        }));
+            }
+        }
```
Comment (MarkFCain) [inline src/SFA.DAS.ApplyService.Data/AssessorRepository.cs:190]: I think you need to add 'AND apply.DeletedAt IS NULL' to both of these

File: src/SFA.DAS.ApplyService.Database/Tables/Apply.sql [hunk 2]
```diff
@@ -24,6 +24,7 @@
     [Assessor1ReviewStatus] NVARCHAR(20) NULL, 
     [Assessor2ReviewStatus] NVARCHAR(20) NULL,
     [NotRequiredOverrides] NVARCHAR(MAX) NULL, 
+    [ModerationStatus] NVARCHAR(20) NOT NULL DEFAULT 'New', 
     [OversightStatus] NVARCHAR(20) NOT NULL DEFAULT 'New', 
     [ApplicationDeterminedDate] DATETIME2 NULL
```
Comment (MarkFCain) [inline src/SFA.DAS.ApplyService.Database/Tables/Apply.sql:29]: probably a good idea to get this addition in a little branch of its own and get it into master ASAP so it doesn't break test releases of other branches

File: src/SFA.DAS.ApplyService.Data/AssessorRepository.cs (hunk 1 above)
Comment (martin-hornsby) [reply src/SFA.DAS.ApplyService.Data/AssessorRepository.cs:190]: And possibly an ApplicationStatus check to signify moving from Assessor to Moderation

File: src/SFA.DAS.ApplyService.Database/Tables/Apply.sql (hunk 2 above)
Comment (martin-hornsby) [reply src/SFA.DAS.ApplyService.Database/Tables/Apply.sql]: this has been done

**Pull request merged**